src/_check.sh

```diff
@@ -0,0 +1,109 @@
+# Handle running checks on the PlexTrac instance
+# Usage
+#   plextrac check        # run checks against the active installation
+#   plextrac check --pre  # run pre-install checks
+
+function mod_check() {
+  if [ ${DO_PREINSTALL_CHECKS:-0} -eq 1 ]; then
+    title "Running pre-installation checks"
+    _check_system_meets_minimum_requirements
+    _check_no_existing_installation
+  else
+    title "Running checks on installation at '${PLEXTRAC_HOME}'"
+    requires_user_plextrac
+    info "Checking Docker Compose Config"
+    compose_client config -q && info "Config check passed"
+    pending=`composeConfigNeedsUpdated || true`
+    if [ "$pending" != "" ]; then
+        error "Pending Changes:"
+        msg "    %s\n" "$pending"
+    fi
+    VALIDATION_ONLY=1 configure_couchbase_users
```

Comment:
existing logic that I wrote, but I don't like it.

This _should_ be a standalone function that checks for invalid/unconfigured users. Then we can either print it (as we do here) or go back and "fix" them (as the function currently behaves if the validation only flag is not set)